webviewer/src/components/Areas/Sidebar/CreateRequest/CreateRequest.jsx

```diff
@@ -41,7 +41,8 @@ export const CreateRequest = ({ areas, currentArea }) => {
         value: layer.id,
         additional_parameter: layer.additional_parameter,
         period_required: layer.period_required,
-        date_type: layer.date_type
+        date_type: layer.date_type,
+        title: `Price ${layer.basic_price} $ per 1 sq. km.`
```

Comment:
maybe better to add layer.basic_price || 0 to avoid undefined if API will miss the bacic_price key